### ISSUE TITLE
The Cat sample fails on response iteration using 4o

### ISSUE THREAD

**Issue by robconery** [microsoft/vscode#224497]
### Extension sample

chat-sample

### VS Code version

1.9.1

### What went wrong?

I'm updating the PG chat extension to use `gpt-4o` and when the model changes, it causes an error while iterating the response. The error does not happen with `gpt-4`.

The exact line is here:

https://github.com/microsoft/vscode-extension-samples/blob/main/chat-sample/src/extension.ts#L37

The exact error text is:

```
TypeError: Cannot read properties of undefined (reading 'convert')
    at Y2._provideLanguageModelResponse (/Users/rob/.vscode/extensions/github.copilot-chat-0.17.1/dist/extension.js:1115:3707)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) {name: 'TypeError', message: "Cannot read properties of undefined (reading 'convert')", stack: 'TypeError: Cannot read properties of undefine…ions (node:internal/process/task_queues:95:5)'}
```

Note: I'm not on pre-release.

**Comment by roblourens** [comment]
Oh, it looks like the Copilot Chat 0.17 extension only supports certain models with a hardcoded set of per-model options. It looks like this has already been changed in the latest, so that it shouldn't happen again next time that a new model is added. Sorry for the inconvenience, but since we've released the new vscode and extension, we'll have to just say that 4o is only supported in the latest release.
